veracode/submit_artifacts.py

```diff
@@ -362,16 +362,15 @@ def submit_artifacts(  # pylint: disable=too-many-return-statements, too-many-br
     return True
 
 
-# pylint: disable=too-many-branches
 @validate
 def is_ready_for_new_build(*, upload_api: UploadAPI) -> bool:
     """
-    Return whether a new build can be started
+    Return whether a build already exists
```

Comment:
Whatever this is should match with the function name

Reply:
Renamed in latest commits.

Reply:
What about just `build_exists` since it returns a `bool`?

Reply:
Moved comment to [here](https://github.com/SeisoLLC/easy_sast/pull/33/commits/28431d761cc05d382a887cec5d8d85fdaf950272#r477514611)